sandpack-react/package.json

```diff
@@ -39,7 +39,7 @@
     "lint": "eslint '**/*.ts?(x)' --fix",
     "build": "rollup -c",
     "build:publish": "yarn build",
-    "start": "tsc -p tsconfig.esm.json --watch",
+    "start": "tsc -p tsconfig.json --watch",
```

Comment:
Interesting! I think this command is not used anywhere and we could even remove it